Dockerfile

```diff
@@ -1,2 +1,4 @@
 FROM node:4-onbuild
+RUN npm install webpack -g
```

Comment:
You should use the most recent version of node so you can take advantage of modern features.